pydarn/utils/conversions.py

```diff
@@ -128,3 +128,46 @@ def dmap2dict(dmap_records: List[dict]) -> List[dict]:
                      for field, data in dmap_record.items()}
         dmap_list.append(OrderedDict(dmap_dict))
     return dmap_list
+
+
+def gate2slant(record, nrang, center=True):
+    """
+    Calculate the slant range (km) for each range gate for SuperDARN data
+
+    Parameters
+    ----------
+        record: dict
+            dictionary of superdarn data records
+        nrang: int
+            max number of range gates in the list of records
+        center: boolean
+            Calculate the slant range in the center of range gate
+            or edge
+
+    Returns
+    -------
+        slant_ranges : np.array
+            returns an array of slant ranges for the radar
+    """
+
+    # lag to the first range gate in microseconds
+    # TODO: with 2.0 and 0.3 what do these values mean?
```

Comment:
I think the 2.0 refers to 'there and back' so the distance is doubled and the 0.3 is the speed of light in microseconds?

Reply:
From units anyway lag(micro second) = frang (m) * 2.0 (unitless) / sol (m / microsecond).  

m's cancel leaving microsecond left.

Reply:
Although I'm not sure 0.3 m/microsecond is the correct value? 
If the units are micro then c=300m/microsecond?
Is lag measured in nanosecond?

Reply:
Never mind! It's in km right? That checks out!
lag(micro second) = frang (km) * 2.0 (unitless) / 0.3 (km / microsecond)

Reply:
so it looks good just will add 0.3 speed of light and 2 is for there and back. 

Should 0.3 be more accurate?

Reply:
I don't think you need it more accurate. It would be 0.2997..., which is going to give a very minimal difference in the actual slant range value you get imo.